src/routes.tsx

```diff
@@ -11,21 +9,11 @@ import AppearanceSettings from '@/components/settings/AppearanceSettings'
 import NotificationsSettings from '@/components/settings/NotificationsSettings'
 import SettingsLayout from '@/components/settings/SettingsLayout'
 import SupportSettings from '@/components/settings/SupportSettings'
-import W3iContext from '@/contexts/W3iContext/context'
 import Login from '@/pages/Login'
 import ScanQrCode from '@/pages/ScanQrCode'
-import WidgetConnect from '@/pages/widget/Connect'
-import WidgetSubscribe from '@/pages/widget/Subscribe'
 
 const ConfiguredRoutes: React.FC = () => {
-  const { uiEnabled } = useContext(W3iContext)
-
-  const defaultPage =
-    Object.entries({
-      '/messages': uiEnabled.chat,
-      '/notifications': uiEnabled.notify,
-      '/settings': uiEnabled.settings
-    }).find(([_, enabled]) => enabled)?.[0] ?? '/login'
+  const defaultPage = '/notifications'
```

Comment:
Same with prior `NAVIGATION` comment. Or we could handle with other opened PR #476